lib/c-api/Cargo.toml

```diff
@@ -32,7 +32,7 @@ wasmer-middlewares = { version = "=3.1.0", path = "../middlewares", optional = t
 wasmer-wasi = { version = "=3.1.0", path = "../wasi", default-features = false, features = ["host-fs", "sys"], optional = true }
 wasmer-types = { version = "=3.1.0", path = "../types" }
 wasmer-vfs = { version = "=3.1.0", path = "../vfs", optional = true, default-features = false, features = ["static-fs"] }
-webc = { version = "3.0.1", optional = true }
+webc = { version = "0.4.1", optional = true }
```

Comment:
Really 3.0.1 -> 0.4.1?

Reply:
Yeah, this was a mistake from when I thought we should match the webc version to the wasmer version

Reply:
Yes, `0.4.1` is the latest version.

Downgrading a major version of a dependency is still a breaking change.

We are stuck in an awkward position here.

We can publich a `3.x` with the latest changes, but that would probably violate semver for `webc`.

Or we can publish a `4.x` webc and upgrade Wasmer to that, but technically a dependency major version upgrade is a breaking change.

Reply:
I vote for an upgrade to 4.x. we cannot stay in this situation.

Reply:
I have asked Felix to release `4.0` .

Reply:
yeah i think 4.0 is a better way forward

Reply:
@theduke it's released now: https://crates.io/crates/webc/4.0.0

Reply:
Was released as 4.0, and the corresponding Cargo.toml files were updated.

NOTE: need to remember that we need a breaking change.